app/models/customer.rb

```diff
@@ -75,6 +75,36 @@ def require_driver_form?
     !driver_form_content.blank?
   end
 
+  # returns Human readable string of the validity length of driver training
+  def helper_driver_training_validity_length?
+    if driver_training_validity_length <= 90
+	return "90 days"
+    elsif driver_training_validity_length <= 180
+        return "180 days"
+    elsif driver_training_validity_length <= 365
+        return "1 year"
+    elsif driver_training_validity_length <= 730
+        return "2 years"
+    else 
+        return "Forever"
+   end
+  end
+
+  # returns human readable string of the validity length of background checks
+  def helper_background_check_validity_length?
+    if background_check_validity_length <= 90
```

Comment:
If you look at these two methods, you can see they share a lot of the same code. You might think about adding a `protected` helper method that each of these two methods call, because they use the same logic. Then if you need to make an adjustment to the logic, you just change the one helper method and both of these methods are updated.